test/e2e/lifecycle/upgrade_test.go

```diff
@@ -17,6 +17,7 @@ var _ = Context("Cluster Network Addons Operator", func() {
 	testUpgrade := func(oldRelease, newRelease Release) {
 		Context(fmt.Sprintf("when operator in version %s is installed and supported spec configured", oldRelease.Version), func() {
 			BeforeEach(func() {
+				UninstallRelease(newRelease)
```

Comment:
question: Why it is needed to uninstall newRelease here? Where it is being installed in the first place?

Reply:
the test is initiated after performing make cluster up+sync, i.e. in the moment of starting the test, the new release is already deployed.
if we try to do that we'll get an error, since downgrade process is not supported.

Reply:
> the test is initiated after performing make cluster up+sync, i.e. in the moment of starting the test, the new release is already deployed.
> if we try to do that we'll get an error, since downgrade process is not supported.

Thank you.
/lgtm